### PR TITLE
feat(FormSelect): added support for placeholder styled option

### LINKED ISSUE #5335: Add placeholder text to Form select
Want to provide placeholder styled text for the Form Select similar to the placeholder styling for other form elements (e.g. Text inputs).

https://github.com/patternfly/patternfly/pull/3790

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #5335

### CODE REVIEW

**Review by redallen — APPROVED** [review]

**Review by mcarrano — APPROVED** [review]
Looks good @tlabaj !

**Review by mcoker — DISMISSED** [review]
🤠👍

File: packages/react-core/src/components/FormSelect/FormSelect.tsx [hunk 1]
```diff
@@ -64,14 +64,18 @@ export class FormSelect extends React.Component<FormSelectProps, { ouiaStateId:
 
   render() {
     const { children, className, value, validated, isDisabled, isRequired, ouiaId, ouiaSafe, ...props } = this.props;
+    /* find sekected option and get placeholder flag */
```
Comment (evwilkin) [inline packages/react-core/src/components/FormSelect/FormSelect.tsx:67]:
```suggestion
    /* find selected option and get placeholder flag */
```

File: packages/react-core/src/components/FormSelect/FormSelectOption.tsx [hunk 2]
```diff
@@ -9,13 +9,17 @@ export interface FormSelectOptionProps extends Omit<React.HTMLProps<HTMLOptionEl
   label: string;
   /** flag indicating if the option is disabled */
   isDisabled?: boolean;
+  /* flag indicating if option will have placeholder styling applied when selected*/
```
Comment (evwilkin) [inline packages/react-core/src/components/FormSelect/FormSelectOption.tsx:12]:
```suggestion
  /* flag indicating if option will have placeholder styling applied when selected */
```
This missing space is causing this text not to show up for prop description on the site.

**Review by evwilkin — CHANGES_REQUESTED** [review]
Functionality looks great, just 2 typos in the documentation

**Review by evwilkin — APPROVED** [review]
LGTM

**Review by redallen — APPROVED** [review]

**Review by dlabrecq — APPROVED** [review]
LGTM

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**